crate_universe/private/util.bzl

```
@@ -48,6 +48,9 @@ def _query_cpu_architecture(repository_ctx, expected_archs, is_windows = False):
     else:
         arch = result.stdout.strip("\n")
```

Small nit: Could we nest the check in the else statement here?
```suggestion
        arch = result.stdout.strip("\n")
        
        # Correct the arm architecture for macos
        if "mac" in repository_ctx.os.name and arch == "arm64":
            arch = "aarch64"

```